docs/_docs/user-guide/eldritch.md

```diff
@@ -312,6 +319,24 @@ crypto.to_json({"foo": "bar"})
 "{\"foo\":\"bar\"}"
 ```
 
+### crypto.md5
+
+`crypto.md5(data: Bytes) -> str`
+
+The <b>crypto.md5</b> method calculates the MD5 hash of the provided data and returns it as a hexadecimal string.
+
+### crypto.sha1
```

Comment:
Remove this, this is not a function we have implemented. Please refer to eldritch-libcrypto/lib.rs for the list of functions.